app/backend/src/couchers/jobs/handlers.py

```diff
@@ -130,6 +137,61 @@ def process_generate_message_notifications(payload):
             )
 
 
+def process_event_creation_emails(payload):
+    """
+    Sends out emails to all users subscribed to a newly created event's cluster.
+    """
+    logger.info(f"Sending out emails for event_id = {payload.event_id}")
+
+    # don't send emails to global and regional events
+    excluded_cluster_parent_node_ids = {
+        "Global Community": 1,
+        "Americas": 4,
+        "Asia-Pacific": 7,
+        "Europe, Africa, and Middle East": 10,
+    }
```

Comment:
These shouldn't be hardcoded here, we can add a db flag or something

Reply:
I think the region IDs can definitely be removed since the logic around maximum notified users kinda handles these same constraints. The global ID makes sense for preventing people from being notified about virtual events, which I understand are always global. How can we include a db flag for that still?